src/gui/sdlmain.cpp

```diff
@@ -4939,7 +4997,7 @@ int sdl_main(int argc, char* argv[])
 		}
 
 		// Timer is needed for title bar animations
-		if (SDL_Init(SDL_INIT_AUDIO | SDL_INIT_VIDEO | SDL_INIT_TIMER) < 0) {
+		if (!SDL_Init(SDL_INIT_AUDIO | SDL_INIT_VIDEO)) {
```

Comment:
SDL 3 documentation recommends to provide some application metadata before `SDL_Init`:
- https://wiki.libsdl.org/SDL3/SDL_SetAppMetadata
- https://wiki.libsdl.org/SDL3/SDL_SetAppMetadataProperty

They should probably replace some legacy calls:
- `SDL_SetHint(SDL_HINT_APP_ID, DOSBOX_APP_ID);`
- `setenv("SDL_VIDEO_X11_WMCLASS", DOSBOX_APP_ID, overwrite);`
- `setenv("SDL_VIDEO_WAYLAND_WMCLASS", DOSBOX_APP_ID, overwrite);`
- `SDL_SetHint(SDL_HINT_APP_NAME, DOSBOX_NAME);`
- `SDL_SetHint(SDL_HINT_AUDIO_DEVICE_STREAM_NAME, DOSBOX_NAME);`